spacy_llm/pipeline/llm.py

```diff
@@ -34,10 +34,8 @@ class CacheConfigType(TypedDict):
     assigns=[],
     default_config={
         "task": None,
-        "backend": {
-            "@llm_backends": "spacy.REST.v1",
-            "api": "OpenAI",
-            "config": {"model": "gpt-3.5-turbo"},
+        "model": {
+            "@llm_models": "spacy.gpt-3-5.v1",
             "strict": True,
```

Comment:
The model `name` isn't here? I would specify the default here for clarity.

Reply:
This is making me reconsider whether the individual models should have a default value for their `name: Literal` setting 🤔 

(also fine by me to think about this in a follow up PR)

Reply:
I'll specify `name` for clarity here. As for the default value - pro: less config, con: less explicit. I'm good with either. I'll leave it up to a follow-up PR to change that.